docs/docs/cloud/how-tos/use_stream_react.md

```diff
@@ -0,0 +1,408 @@
+# How to stream runs in your React frontend
+
+!!! info "Prerequisites"
+    - [LangGraph Platform](../concepts/langgraph_platform.md)
+    - [LangGraph Server](../concepts/langgraph_server.md)
+
+The `useStream()` React hook provides a seamless way to integrate LangGraph runs into your React applications. It handles all the complexities of streaming, state management, and branching logic, letting you focus on building great chat experiences.
+
+Key features:
+
+- Messages streaming: Handle a stream of message chunks to form a complete message
+- Automatic state management for messages, loading states, and errors
+- Conversation branching: Create alternate conversation paths from any point in the chat history
+- UI-agnostic design - bring your own components and styling
+
+Let's explore how to use `useStream()` in your React application.
+
+## Example
+
+```tsx
+"use client";
+
+import { useStream } from "@langchain/langgraph-sdk/react";
+import type { Message } from "@langchain/langgraph-sdk";
+
+export default function App() {
+  const thread = useStream<{ messages: Message[] }>({
+    apiUrl: "http://localhost:2024",
+    assistantId: "agent",
+    messagesKey: "messages",
```

Comment:
Can we make this default to `messages`?

Reply:
And document that by default it'll look for a key in emitted chunks named `messages`

Reply:
https://github.com/langchain-ai/langgraph/pull/3425

Reply:
Nice